cmd/cluster/health.go

```diff
@@ -140,15 +139,7 @@ func (o *healthOptions) run() error {
 		}
 	}
 
-	// Extracting region from the availability zone.
-	reg := cluster.Nodes().AvailabilityZones()[0]
-	length := len(reg)
-	lastChar := reg[length-1 : length]
-	for _, r := range lastChar {
-		if unicode.IsLetter(r) {
-			reg = reg[0 : length-1]
-		}
-	}
+	reg := cluster.Region().ID()
```

Comment:
Note sure we can replace the availability zone with the region id, here in staging on some random cluster:
```
ngrauss@ngrauss-mac rosa % ocm get cluster 1shn6cg32l3eb2bao4tp2hl41h8rffng | jq .region.id               
"us-east-2"
ngrauss@ngrauss-mac rosa % ocm get cluster 1shn6cg32l3eb2bao4tp2hl41h8rffng | jq .nodes.availability_zones
[
  "us-east-2a"
]
```

Reply:
Ok FINALLY managed to find a prod example as well:
```
ngrauss@ngrauss-mac rosa % ocm get cluster 1m43aku52bcv9808vb0ga612r5hg51sm | jq .nodes.availability_zones
[
  "eu-west-1a",
  "eu-west-1b",
  "eu-west-1c"
]
ngrauss@ngrauss-mac rosa % ocm get cluster 1m43aku52bcv9808vb0ga612r5hg51sm | jq .region.id               
"eu-west-1"
```

Reply:
As you can see the availability sone contains the some letter at the end because a region is sliced in availability zones.
Hopes this helps.